lisp/forge-commands.el

```diff
@@ -931,6 +936,14 @@ heavy development."
     (emacsql-enable-debugging db)
     (switch-to-buffer-other-window (emacsql-log-buffer db))))
 
+(defvar-local forge-display-sections t
+  "Whether the Forge sections are inserted in the Magit status buffer upon refresh.")
+
+;;;###autoload
+(defun forge-toggle-display-sections ()
+  (interactive)
+  (setq forge-display-sections (if forge-display-sections nil t)))
```

Comment:
This should be documented and maybe you can think of a better name.

Reply:
You added a doc-string to the variable, but what really needs one is this command. :grinning:

Reply:
Oh and to flip a boolean you can just use `not` around the current value.

Reply:
Hope I got the right doc-string this time :)